services/editor/src/components/JPadFullEditor/JPadVisualEditor/Matcher/Properties/styles.css

```diff
@@ -1128,6 +1128,7 @@
   margin-right: 8px;
   display: inline-block;
   align-self: center;
+  z-index: 2;
 }
 .property-combobox-container .avatar-container.property-avatar {
   zoom: 75%;
```

Comment:
along the way, please replace it with `transform: scale`

Reply:
didn't do it because of ui issues